examples/spark/images/base/Dockerfile

```diff
@@ -3,15 +3,35 @@ FROM java:latest
 RUN apt-get update -y
 RUN apt-get install -y scala
 
-# Get Spark from some apache mirror.
+ENV hadoop_ver 2.6.1
+ENV spark_ver 1.5.1
+
+# Get Hadoop from US Apache mirror and extract just the native
+# libs. (Until we care about running HDFS with these containers, this
+# is all we need.)
 RUN mkdir -p /opt && \
     cd /opt && \
-    wget http://apache.mirrors.pair.com/spark/spark-1.4.0/spark-1.4.0-bin-hadoop2.6.tgz && \
-    tar -zvxf spark-1.4.0-bin-hadoop2.6.tgz && \
-    rm spark-1.4.0-bin-hadoop2.6.tgz && \
-    ln -s spark-1.4.0-bin-hadoop2.6 spark && \
-    echo Spark installed in /opt
+    wget http://www.us.apache.org/dist/hadoop/common/hadoop-${hadoop_ver}/hadoop-${hadoop_ver}.tar.gz && \
```

Comment:
Why are you pulling in hadoop?  The versions you download are prebuilt against hadoop2.6.